src/Panel.tsx

```diff
@@ -4,7 +4,11 @@ import { AddonPanel } from "@storybook/components";
 import { ADDON_ID, EVENTS } from "./constants";
 import { PanelContent } from "./components/PanelContent";
 
-export const Panel = (props) => {
+type PanelProps = {
```

Comment:
maybe `interface`? i think TS people prefer them although I don't know the rationale. cc @ndelangen

Reply:
Looks like we use interface everywhere else. I'll use that.